ertenclave/structs.h

```diff
@@ -0,0 +1,85 @@
+#define OE_UNIQUE_ID_SIZE 32
```

Comment:
```suggestion
// Copyright (c) Open Enclave SDK contributors.
// Licensed under the MIT License.

#define OE_UNIQUE_ID_SIZE 32
```
we should better keep the license notice